roles/wildfly_app_deploy/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Path to the application artifact (.war, .ear, .jar) on the Ansible controller
+wildfly_app_deploy_file: "/path/to/your/application.war"
+
+# The name to use for the deployment in WildFly (e.g., app-v1.war)
+wildfly_app_deploy_name: "application.war"
+
+# Deployment mode: 'standalone' or 'domain'
+wildfly_app_deploy_mode: "standalone"
+
+# List of server groups to deploy to (Required if wildfly_app_deploy_mode is 'domain')
+# Example: ['main-server-group', 'other-server-group']
+wildfly_app_deploy_server_groups: []
```

Comment:
I think this variable could be renamed to `wildfly_app_deploy_domain_server_groups` to make it clearer